backend/src/main/java/gov/cdc/usds/simplereport/idp/repository/LiveOktaRepository.java

```diff
@@ -175,7 +180,7 @@ public Optional<OrganizationRoleClaims> createUser(
     Set<String> groupIdsToAdd =
         orgGroups.stream()
             .filter(g -> groupNamesToAdd.contains(g.getProfile().getName()))
-            .map(g -> g.getId())
+            .map(Group::getId)
```

Comment:
for curiosity, non-blocking - is there a reason we prefer using the double colon here rather than the lambda? Wondering if this is just a style thing or if there are other benefits for passing the function like this

Reply:
I don't believe there are any actual benefits other than style. If the team prefers it the other way, I'll be happy to change it back!

Reply:
this is probably a bikeshedding point / not worth the effort to change it, so we can leave it as is